src/EnergyPlus/Psychrometrics.hh

```diff
@@ -1343,4 +1376,4 @@ namespace Psychrometrics {
 
 } // namespace EnergyPlus
 
-#endif
+#endif
```

Comment:
This file needs an endline at the end.

Reply:
I'll probably just add this to one of my upcoming globals branches and not spend more CI cycles on this.

Reply:
:+1: